src/plays/git-hub-profile-search/GithubProfileSearch.js

```diff
@@ -0,0 +1,36 @@
+import PlayHeader from 'common/playlists/PlayHeader';
+import './styles.css';
+import Input from './components/Input';
+import Results from './components/Results';
+import { ResultContextProvider } from './context/ResultContextProvider';
+
+// WARNING: Do not change the entry componenet name
+function GithubProfileSearch(props) {
+
+  // Your Code Start below.
+
+  return (
+    <>
+      <div className="play-details">
+        <PlayHeader play={props} />
+        <div className="play-details-body">
```

Comment:
Add a unique class name for ur play and use that as a prefix for all the styles in the CSS. It will not impact the other plays.

Reply:
changed and added new ones

Reply:
@nirban256 I don't see this change.

Reply:
these were present automatically and changing those changes the styling and alignment of the header.

Reply:
I am not asking you to change the existing ones. They should how they are. Add an additional class name along with `play-details-body`, say, `nirban-github-users-search`. Now you can use this new class name as. the prefix to all your styles in the CSS file.

Reply:
okay I will add those

Reply:
made the change  @atapas , I am not using those in css as the styling is already done.

Reply:
@nirban256 then what’s the use of adding it 😀? I asked you to add it so that you can use it as the prefix to all your styles. That way none of your styles  conflicts.

you just need to put this new class in the front of every styles defined in CSS.

Reply:
is it mandatory to do because I have already did the styling @atapas

Reply:
@atapas I have made the changes